packages/next-app/package.json

```diff
@@ -35,6 +37,7 @@
     "react-dom": "^18.2.0",
     "react-hook-form": "^7.43.5",
     "swr": "^2.1.0",
+    "react-icons": "^4.8.0",
```

Comment:
didn't you find the icons you need on radix's library?

Reply:
Sure... switched back to radix-ui/react-icons ✌🏻
cc: @devjoaov